Tasks/ServiceFabricDeploy/task.json

```diff
@@ -270,6 +270,15 @@
             "required": false,
             "groupname": "upgrade",
             "visibleRule": "overridePublishProfileSettings = true && isUpgrade = true && upgradeMode = Monitored"
+        },
+        {
+            "name": "skipUpgradeSameTypeAndVersion",
+            "type": "boolean",
+            "label": "Skip upgrade for same Type and Version",
+            "defaultValue": "false",
+            "required": false,
+            "groupname": "upgrade",
+            "helpMarkDown": "Indicates whether an upgrade will be skipped if the same application type and version already exists in the cluster, otherwise the upgrade fails during validation. If enabled, re-deployments are idempotent."
```

Comment:
Once I deployed this I noticed the UI around this check-box gets a bit funky with the behavior of the "Override All Publish Profile Upgrade Settings" check-box. could you please move this to the advanced group.